### ISSUE TITLE
TC Score calculation logic is implemented and validated on sample data.

### ISSUE THREAD

**Issue by itay55** [arielba2002/Deep-Picker-Project#18]

**Comment by BenEpstein** [comment]
@guygordon 

@bazakeliad and myself, created the DeepPicker-DataScraping.ipynb notebook in which we scraped players data into a raw_data.json file. 

We left a placeholder `labels: []` inside for your team stats scraping values. (TC score) 

If you have any improvement ideas or modifications you would like to make. Please feel free to do so.

**Comment by guygordon** [comment]
I'm not sure what do you mean by not using tc score as label? 
The variables that you mentioned make the tc score for training, their combination eventually adjusts the tc score so later it could be generated in inference.

**Comment by BenEpstein** [comment]
> @BenEpstein 
> I'm not sure what do you mean by not using tc score as label? 
> The variables that you mentioned make the tc score for training, their combination eventually adjusts the tc score so later it could be generated in inference.

Yes exactly, I think the TC score should be generated in inference and not in the dataset preprocessing stage.
That would mean we need to predict multiple team evaluation stats at the training stage.

**Comment by bazakeliad** [comment]
Currently, for POC we aggregate all teams' season metrics and refer to the resulting sum as the label.

**Comment by BenEpstein** [comment]
Closing as POC is done.

**Comment by BenEpstein** [comment]
New tasks will be opened relevant to final project.
